lib/src/ast/attribute.dart

```diff
@@ -13,10 +13,7 @@ import 'package:quiver/core.dart';
 /// Clients should not extend, implement, or mix-in this class.
 abstract class AttributeAst implements TemplateAst {
   /// Create a new synthetic [AttributeAst] with a string [value].
-  factory AttributeAst(
-    String name, [
-    String value,
-  ]) = _SyntheticAttributeAst;
+  factory AttributeAst(String name, [String value]) = _SyntheticAttributeAst;
```

Comment:
This is a relatively small formatting point, but do you mind reverting the changes removing all the trailing commas here and across the PR? I realize it's overkill for 1-2 parameters, but because there are some classes with many, I'd prefer them _all_ to have trailing commas for consistency.

I imagine it also will make the delta of this CL much smaller.